src/i18n.ts

```diff
@@ -94,6 +94,9 @@ const loadedLocales = new Set<string>(['en'])
 // Track locales currently being loaded to prevent race conditions
 const loadingLocales = new Map<string, Promise<void>>()
 
+// Store custom nodes i18n data for merging when locales are lazily loaded
+let customNodesI18nData: Record<string, Record<string, unknown>> = {}
```

Comment:
For a module level property like this, can we make the record itself a `const`?
I want to avoid the possibility of older versions being held onto.

Also, is there a stricter type than `unknown` that we could enforce?

Reply:
we use any in api.ts, see https://github.com/Comfy-Org/ComfyUI_frontend/blob/f80654ae31da97745d6e67e31c254b43db794264/src/scripts/api.ts#L1257
to keep consistency as api.ts, I will change it to any as type